script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1_1.txt

```diff
@@ -22,21 +22,21 @@ I resist the urge to say, "No, seven o'clock is normal. This house is not normal
 <Page8>
 "Hmm. What time do you get up, Akiha?"
 "I wake up at five o'clock, what about it?"
-"...I'm sorry, My mistake."
+"...I'm sorry, my mistake."
 Despite saying something that would be unthinkable for a normal person, she is wearing an expression that says "is that so strange?". I can't help but be dumbfounded.
-I've never seen a more perfect response from her.
+I've never heard such a perfect response.
 <Page9>
-"? What did you mean by "your mistake"?"
-"No, nothing...but what are you doing up so early? Trying not to be late?"
+"What do you mean your 'mistake'?"
+"Ah... Nevermind that, what are you doing up so early? Trying not to be late?"
 <Page10>
-"Listen here. Unless there is a serious problem, I will not be late. That will never happen.
-Your school is about a half-hour walk from here, right? It is not that far, so do not be late. It would be embarrassing if you were."
+"Listen here. Unless something serious happens, I will not be late. That will never happen.
+Your school is about a half-hour walk from here, right? It's not that far, so do not be late. It would be embarrassing if you were."
 <Page11>
-Well, her thorny words are really sharp and prickly.
-However, all of her points are correct, and I can't think of any objections.
-It's natural for Akiha to be concerned about the reputation of the Tohno family, no, as the head of the family, she must fulfill it.
+Well, she's prickly as ever.
```

Comment:
the original translation is more accurate (see jp), but sounds super weird. This is closer to what you'd actually say in English.
> 棘があってチクチクする